src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/wwwroot/service-worker.published.js

```diff
@@ -42,13 +36,7 @@ async function onFetch(event) {
     let cachedResponse = null;
     if (event.request.method === 'GET') {
         // For all navigation requests, try to serve index.html from cache
-        // If you need some URLs to be server-rendered, edit the following check to exclude those URLs
-//#if(Hosted)
-        const shouldServeIndexHtml = event.request.mode === 'navigate'
-            && !event.request.url.includes('/connect/')
-//#else
         const shouldServeIndexHtml = event.request.mode === 'navigate';
-//#endif
```

Comment:
Is this related to authentication? Wondering why this may be necessary (possibly due to the `/connect/` endpoint?)

Reply:
This was related to authentication, but this was modified to remove `IndividualLocalAuth` after being transferred from the default template. The `/connect/` endpoint is not really necessary in this case.

This is the original if statement in `/ComponentsWebAssembly-CSharp/Client/wwwroot/service-worker.published.js`:

```csharp
//#if(IndividualLocalAuth && Hosted)
        const shouldServeIndexHtml = event.request.mode === 'navigate'
            && !event.request.url.includes('/connect/')
            && !event.request.url.includes('/Identity/');